src/main/java/de/jollyday/util/Cache.java

```diff
@@ -37,7 +37,10 @@
 	 * @return the eventually cached value
 	 */
 	public VALUE get(ValueHandler<VALUE> valueHandler) {
-		return cachingMap.computeIfAbsent(valueHandler.getKey(), k -> valueHandler.createValue());
+		final String key = valueHandler.getKey();
+		// Try to first get the value which is most likely cached to avoid creating a lambda.
+		final VALUE value = cachingMap.get(key);
+		return value != null ? value : cachingMap.computeIfAbsent(key, k -> valueHandler.createValue());
```

Comment:
@svendiedrichsen can you see this comment? I know where comments are made in the code, but trust me when I tell you that I cannot see your comments.